bioimageio_core_version: 0.5.9
bioimageio_spec_version: 0.4.9
rdf_sha256: 7a6002923c5aa5eaf2f61ff2849044a62fee325f4b2f4e3075ad2086ef00d505
status: passed
tests:
  bioimageio:
  - bioimageio_spec_version: 0.4.9
    error: null
    name: bioimageio.spec static validation of model RDF 0.4.9 with update to latest
      format version
    nested_errors: {}
    status: passed
    traceback: null
    warnings: {license: BSD-2-Clause (BSD 2-Clause "Simplified" License) is not FSF
        Free/libre., tags: 'Missing tags corresponding to bioimage.io categories:
        [{''modality'': [''electron-microscopy'', ''cryo-electron-microscopy'', ''fluorescence-light-microscopy'',
        ''transmission-light-microscopy'', ''super-resolution-microscopy'', ''x-ray-microscopy'',
        ''force-microscopy'', ''high-content-imaging'', ''whole-slide-imaging'', ''other'']},
        {''content'': [''cells'', ''nuclei'', ''extracellular-vesicles'', ''tissue'',
        ''plant'', ''mitochondria'', ''vasculature'', ''cell-membrane'', ''brain'',
        ''whole-organism'']}, {''task'': [''semantic-segmentation'', ''instance-segmentation'',
        ''object-detection'', ''image-classification'', ''denoising'', ''image-restoration'',
        ''image-reconstruction'', ''in-silico-labeling'']}]'}
  - bioimageio_spec_version: 0.4.9
    error: null
    name: bioimageio.spec static validation of model RDF 0.4.9
    nested_errors: {}
    status: passed
    traceback: null
    warnings: {license: BSD-2-Clause (BSD 2-Clause "Simplified" License) is not FSF
        Free/libre., tags: 'Missing tags corresponding to bioimage.io categories:
        [{''modality'': [''electron-microscopy'', ''cryo-electron-microscopy'', ''fluorescence-light-microscopy'',
        ''transmission-light-microscopy'', ''super-resolution-microscopy'', ''x-ray-microscopy'',
        ''force-microscopy'', ''high-content-imaging'', ''whole-slide-imaging'', ''other'']},
        {''content'': [''cells'', ''nuclei'', ''extracellular-vesicles'', ''tissue'',
        ''plant'', ''mitochondria'', ''vasculature'', ''cell-membrane'', ''brain'',
        ''whole-organism'']}, {''task'': [''semantic-segmentation'', ''instance-segmentation'',
        ''object-detection'', ''image-classification'', ''denoising'', ''image-restoration'',
        ''image-reconstruction'', ''in-silico-labeling'']}]'}
  - bioimageio_core_version: 0.5.9
    bioimageio_spec_version: 0.4.9
    error: null
    name: load resource description
    nested_errors: null
    status: passed
    traceback: null
    warnings: {}
  - bioimageio_core_version: 0.5.9
    bioimageio_spec_version: 0.4.9
    error: null
    name: All URLs and paths available
    nested_errors: null
    status: passed
    traceback: null
    warnings: {}
  - bioimageio_core_version: 0.5.9
    bioimageio_spec_version: 0.4.9
    error: null
    name: Test documentation completeness.
    status: passed
    traceback: null
    warnings: {documentation: No '# Validation' (sub)section found.}
  - bioimageio_core_version: 0.5.9
    bioimageio_spec_version: 0.4.9
    error: null
    name: reproduce test outputs from test inputs (bioimageio.core 0.5.9)
    status: passed
    traceback: null
    warnings:
      non-validation-warnings: [Computed dataset percentiles naively by averaging
          percentiles of samples., 'Device management is not implemented for keras
          yet, cannot unload model']
  deepimagej:
  - {bioimageio_spec_version: 0.4.8post1, name: Reproduce test outputs with DeepImageJ
      headless, status: passed}
  ilastik:
  - bioimageio_core_version: 0.5.7
    bioimageio_spec_version: 0.4.8
    error: null
    name: load resource description
    status: passed
    traceback: null
    warnings: {}
  - {error: null, name: has expected resource type, status: passed, traceback: null}
  - bioimageio_core_version: 0.5.7
    bioimageio_spec_version: 0.4.8
    error: null
    name: All URLs and paths available
    nested_errors: null
    status: passed
    traceback: null
    warnings: {}
  - bioimageio_core_version: 0.5.7
    bioimageio_spec_version: 0.4.8
    error: null
    name: Test documentation completeness.
    status: passed
    traceback: null
    warnings: {documentation: No '# Validation' (sub)section found.}
  - bioimageio_core_version: 0.5.7
    bioimageio_spec_version: 0.4.8
    error: null
    name: reproduce test outputs from test inputs
    status: passed
    traceback: null
    warnings:
      non-validation-warnings: ['Model tensorflow version (1, 15) does not match 1.14.0.The
          prediction results may be wrong', 'Device management is not implemented
          for keras yet, cannot unload model', Computed dataset percentiles naively
          by averaging percentiles of samples.]
  - bioimageio_core_version: 0.5.8
    bioimageio_spec_version: 0.4.9
    error: null
    name: load resource description
    nested_errors: null
    status: passed
    traceback: null
    warnings: {}
  - bioimageio_core_version: 0.5.8
    bioimageio_spec_version: 0.4.9
    error: null
    name: All URLs and paths available
    nested_errors: null
    status: passed
    traceback: null
    warnings: {}
  - bioimageio_core_version: 0.5.8
    bioimageio_spec_version: 0.4.9
    error: null
    name: Test documentation completeness.
    status: passed
    traceback: null
    warnings: {documentation: No '# Validation' (sub)section found.}
  - bioimageio_core_version: 0.5.8
    bioimageio_spec_version: 0.4.9
    error: null
    name: reproduce test outputs from test inputs
    status: passed
    traceback: null
    warnings:
      non-validation-warnings: ['Device management is not implemented for keras yet,
          cannot unload model', Computed dataset percentiles naively by averaging
          percentiles of samples., 'Model tensorflow version (1, 15) does not match
          1.14.0.The prediction results may be wrong']
  - bioimageio_core_version: 0.5.8
    bioimageio_spec_version: 0.4.9
    error: '''str'' object has no attribute ''decode'''
    name: reproduce test outputs from test inputs
    status: failed
    traceback: ["  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/resource_tests.py\",
        line 157, in _test_model_inference\n    bioimageio_model=model, devices=devices,
        weight_format=weight_format\n", "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_prediction_pipeline.py\",
        line 122, in __enter__\n    self.load()\n", "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_prediction_pipeline.py\",
        line 180, in load\n    self._model.load()\n", "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_model_adapters/_model_adapter.py\",
        line 57, in load\n    self._load(devices=devices or self.default_devices)\n",
      "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_model_adapters/_keras_model_adapter.py\",
        line 41, in _load\n    self._model = keras.models.load_model(weight_file)\n",
      "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/tensorflow/python/keras/saving/save.py\",
        line 146, in load_model\n    return hdf5_format.load_model_from_hdf5(filepath,
        custom_objects, compile)\n", "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/tensorflow/python/keras/saving/hdf5_format.py\",
        line 210, in load_model_from_hdf5\n    model_config = json.loads(model_config.decode('utf-8'))\n"]
    warnings:
      non-validation-warnings: ['Model tensorflow version (1, 15) does not match 1.14.0.The
          prediction results may be wrong']
  - bioimageio_core_version: 0.5.7
    bioimageio_spec_version: 0.4.8
    error: '''str'' object has no attribute ''decode'''
    name: reproduce test outputs from test inputs
    status: failed
    traceback: ["  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/resource_tests.py\",
        line 157, in _test_model_inference\n    bioimageio_model=model, devices=devices,
        weight_format=weight_format\n", "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_prediction_pipeline.py\",
        line 122, in __enter__\n    self.load()\n", "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_prediction_pipeline.py\",
        line 180, in load\n    self._model.load()\n", "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_model_adapters/_model_adapter.py\",
        line 57, in load\n    self._load(devices=devices or self.default_devices)\n",
      "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/bioimageio/core/prediction_pipeline/_model_adapters/_keras_model_adapter.py\",
        line 41, in _load\n    self._model = keras.models.load_model(weight_file)\n",
      "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/tensorflow/python/keras/saving/save.py\",
        line 146, in load_model\n    return hdf5_format.load_model_from_hdf5(filepath,
        custom_objects, compile)\n", "  File \"/usr/share/miniconda3/envs/ilastik-env/lib/python3.7/site-packages/tensorflow/python/keras/saving/hdf5_format.py\",
        line 210, in load_model_from_hdf5\n    model_config = json.loads(model_config.decode('utf-8'))\n"]
    warnings:
      non-validation-warnings: ['Model tensorflow version (1, 15) does not match 1.14.0.The
          prediction results may be wrong']
